docs(s3cmd): upload with mime type (#6636)
scaleway/docs-content: pages/object-storage/api-cli/managing-lifecycle-cliv2.mdx

    upload: '/path/to/object/my-object' -> 's3://my-bucket/my-object'  [1 of 1]
    3259 of 3259   100% in    0s     8.47 KB/s  done
    ```

    <Message type="tip">

> By default, s3cmd does not send any mime type, and the object will be stored as `plain text`. Using `--guess-mime-type` and `--no-mime-magic`, you can make your bucket provide the file with the mime type associated to the extension. This example will store my-object-.css with a `text/css` content-type instead of `text/plain`.

Reviewer: ```suggestion
Object Storage uses the MIME type sent by s3cmd. If Object Storage is unable to determine the MIME type because the client (in this case s3cmd) has not sent any, then it first tries to infer the correct MIME type from the file extension; and if it cannot find a matching type, it falls back to the default MIME type, which is `binary/octet-stream`.
```

Reviewer: Thank you for your contribution, it is much appreciated. 

I have talked to someone in Dev to help with the technical review of your PR. He suggested changing the Tip as per the suggestions that I made.

Is that OK with you?

Author: ok for me

    ```
    s3cmd --guess-mime-type --no-mime-magic put /path/to/object/my-object.css s3://my-bucket/my-object.css
    ```

>     </Message>

Reviewer 2: ```suggestion
    <Message type="tip">
    By default, s3cmd does not send any mime type, and the object will be stored as `plain text`. Using `--guess-mime-type` and `--no-mime-magic`, you can make your bucket provide the file with the mime type associated to the extension. This example will store my-object-.css with a `text/css` content-type instead of `text/plain`.
    ```
    s3cmd --guess-mime-type --no-mime-magic put /path/to/object/my-object.css s3://my-bucket/my-object.css
    ```
    </Message>
```

Author: Thank you for your review, I updated the 2 files with your suggestions.

>     ```

Reviewer: ```suggestion
```
